MWContentDisplayPlugin/MWContentDisplayPlugin/ContentGridStep/Views/MWImageCollectionViewCell.swift

```diff
@@ -19,96 +17,80 @@ class MWImageCollectionViewCell: UICollectionViewCell {
         let imageUrl: URL?
     }
     
-    var delegate: MWImageCollectionViewCellDelegate?
-    
-    //MARK: - UI
-    private let titleLabel: UILabel!
-    private let subtitleLabel: UILabel!
-    private let imageView: UIImageView!
-    
+    //MARK: Class properties
+    private let titleLabel = UILabel()
+    private let subtitleLabel = UILabel()
+    private let imageView = UIImageView()
     private var imageLoadTask: AnyCancellable?
     
+    //MARK: Lifecycle
     override init(frame: CGRect) {
         
-        let titleLabel = UILabel()
-        titleLabel.translatesAutoresizingMaskIntoConstraints = false
-        titleLabel.setContentCompressionResistancePriority(.required, for: .vertical)
-        titleLabel.setContentHuggingPriority(.required, for: .vertical)
-        titleLabel.font = .preferredFont(forTextStyle: .body, weight: .regular)
-        titleLabel.textColor = .label
-        self.titleLabel = titleLabel
-        
-        let subtitleLabel = UILabel()
-        subtitleLabel.translatesAutoresizingMaskIntoConstraints = false
-        subtitleLabel.setContentCompressionResistancePriority(.required-1, for: .vertical)
-        subtitleLabel.setContentHuggingPriority(.required-1, for: .vertical)
-        subtitleLabel.font = .preferredFont(forTextStyle: .caption1, weight: .regular)
-        subtitleLabel.textColor = .secondaryLabel
-        self.subtitleLabel = subtitleLabel
+        self.titleLabel.translatesAutoresizingMaskIntoConstraints = false
+        self.titleLabel.setContentCompressionResistancePriority(.required, for: .vertical)
+        self.titleLabel.setContentHuggingPriority(.required, for: .vertical)
+        self.titleLabel.font = .systemFont(ofSize: 17, weight: .regular)
```

Comment:
Not possible to support dynamic type for various labels?

Reply:
Let me see if I can make it happen, for regular texts should be easy because the fonts match, for titles might be a bit tricky because they were not following the standard ones if I remember correctly.

Reply:
https://github.com/FutureWorkshops/MWContentDisplayPlugin-iOS/pull/16/commits/85853e8eaba81ab7c0db7323f413b6b3a1912167 removes the hardcoded fonts:

![Image 3](https://user-images.githubusercontent.com/8229382/116227236-36001a80-a754-11eb-8bb6-6fb67ff7bff9.PNG)